src/components/academy/Missions.tsx

```diff
@@ -59,14 +89,16 @@ export const MissionInfoCard: React.SFC<IMissionInfoCardProps> = props => {
               </Text>
             </div>
             <div className="col-xs">
-              <Button
-                className="mission-info-skip-button"
-                minimal={true}
-                intent={Intent.PRIMARY}
-                icon={IconNames.FLAME}
-              >
-                Skip Story & Attempt
-              </Button>
+              <NavLink to={'/academy/missions/' + mission.id.toString()}>
```

Comment:
Is there a a better way a link can be implemented? This requires `MemoryRouter` to be in place for the component test.